concepts/05 Widgets/Form/01 Getting Started with Form/70 Validate the Form Data.md

```diff
@@ -0,0 +1,224 @@
+DevExtreme includes a validation engine that checks edited values before they are saved. This engine supports different validation rule types, such as [Email](/Documentation/ApiReference/UI_Widgets/dxValidator/Validation_Rules/EmailRule/), [Numeric](/Documentation/ApiReference/UI_Widgets/dxValidator/Validation_Rules/NumericRule/), [Pattern](/Documentation/ApiReference/UI_Widgets/dxValidator/Validation_Rules/PatternRule/), and more. 
+
```

Comment:
> and more.

See https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxValidator/Validation_Rules/ 
Can we add this link?

Reply:
we have it in line below